src/main/java/org/altadoon/gt6x/features/verticalmixers/VerticalMixers.java

```diff
@@ -0,0 +1,86 @@
+        mat = MT.DATA.Electric_T[3]; MTEx.gt6xMTEReg.add("Vertical Electric Mixer ("                          +VN[3]+")", "Basic Machines"                      , 113, 20001, aClass, mat.mToolQuality, 16, MTEx.MachineBlock     , UT.NBT.make(NBT_MATERIAL, mat, NBT_HARDNESS,   4.0F, NBT_RESISTANCE,   4.0F, NBT_INPUT,   512, NBT_TEXTURE, "electricmixer_vertical", NBT_ENERGY_ACCEPTED, TD.Energy.EU, NBT_RECIPEMAP, RM.Mixer, NBT_EFFICIENCY, 5000, NBT_INV_SIDE_IN, SBIT_U|SBIT_L, NBT_INV_SIDE_AUTO_IN, SIDE_LEFT, NBT_INV_SIDE_OUT, SBIT_D|SBIT_R, NBT_INV_SIDE_AUTO_OUT, SIDE_RIGHT, NBT_TANK_SIDE_IN, SBIT_U|SBIT_R, NBT_TANK_SIDE_AUTO_IN, SIDE_TOP, NBT_TANK_SIDE_OUT, SBIT_D|SBIT_L, NBT_TANK_SIDE_AUTO_OUT, SIDE_BOTTOM, NBT_ENERGY_ACCEPTED_SIDES, SBIT_B, NBT_PARALLEL, 16, NBT_PARALLEL_DURATION, T));
+        mat = MT.DATA.Electric_T[4]; MTEx.gt6xMTEReg.add("Vertical Electric Mixer ("                          +VN[4]+")", "Basic Machines"                      , 114, 20001, aClass, mat.mToolQuality, 16, MTEx.MachineBlock     , UT.NBT.make(NBT_MATERIAL, mat, NBT_HARDNESS,   4.0F, NBT_RESISTANCE,   4.0F, NBT_INPUT,   2048, NBT_TEXTURE, "electricmixer_vertical", NBT_ENERGY_ACCEPTED, TD.Energy.EU, NBT_RECIPEMAP, RM.Mixer, NBT_EFFICIENCY, 5000, NBT_INV_SIDE_IN, SBIT_U|SBIT_L, NBT_INV_SIDE_AUTO_IN, SIDE_LEFT, NBT_INV_SIDE_OUT, SBIT_D|SBIT_R, NBT_INV_SIDE_AUTO_OUT, SIDE_RIGHT, NBT_TANK_SIDE_IN, SBIT_U|SBIT_R, NBT_TANK_SIDE_AUTO_IN, SIDE_TOP, NBT_TANK_SIDE_OUT, SBIT_D|SBIT_L, NBT_TANK_SIDE_AUTO_OUT, SIDE_BOTTOM, NBT_ENERGY_ACCEPTED_SIDES, SBIT_B, NBT_PARALLEL, 32, NBT_PARALLEL_DURATION, T));
+        mat = MT.DATA.Electric_T[5]; MTEx.gt6xMTEReg.add("Vertical Electric Mixer ("                          +VN[5]+")", "Basic Machines"                      , 115, 20001, aClass, mat.mToolQuality, 16, MTEx.MachineBlock     , UT.NBT.make(NBT_MATERIAL, mat, NBT_HARDNESS,   4.0F, NBT_RESISTANCE,   4.0F, NBT_INPUT,   8192, NBT_TEXTURE, "electricmixer_vertical", NBT_ENERGY_ACCEPTED, TD.Energy.EU, NBT_RECIPEMAP, RM.Mixer, NBT_EFFICIENCY, 5000, NBT_INV_SIDE_IN, SBIT_U|SBIT_L, NBT_INV_SIDE_AUTO_IN, SIDE_LEFT, NBT_INV_SIDE_OUT, SBIT_D|SBIT_R, NBT_INV_SIDE_AUTO_OUT, SIDE_RIGHT, NBT_TANK_SIDE_IN, SBIT_U|SBIT_R, NBT_TANK_SIDE_AUTO_IN, SIDE_TOP, NBT_TANK_SIDE_OUT, SBIT_D|SBIT_L, NBT_TANK_SIDE_AUTO_OUT, SIDE_BOTTOM, NBT_ENERGY_ACCEPTED_SIDES, SBIT_B, NBT_PARALLEL, 64, NBT_PARALLEL_DURATION, T));
+
+
+        CR.shapeless(MTEx.gt6xMTEReg.getItem(111),new Object[]{MTEx.gt6Registry.getItem(20351), OreDictToolNames.wrench});
```

Comment:
personally, I would have just flipped the crafting recipe by 90 degrees, but I guess this one is reversible which is maybe better

Reply:
fixed in [b5c82c6](https://github.com/BorisMulder/gt6x/pull/13/commits/b5c82c63272b95ccee73d677296d3e31cd82eadc)